csharp/src/Ice/ObjectAdapter.cs

```diff
@@ -187,16 +186,14 @@ public void waitForHold()
     }
 
     /// <summary>
-    /// Deactivate all endpoints that belong to this object adapter.
-    /// After deactivation, the object adapter stops
-    /// receiving requests through its endpoints. Object adapters that have been deactivated must not be reactivated
-    /// again, and cannot be used otherwise. Attempts to use a deactivated object adapter raise
-    /// ObjectAdapterDeactivatedException however, attempts to deactivate an already deactivated
-    /// object adapter are ignored and do nothing. Once deactivated, it is possible to destroy the adapter to clean up
-    /// resources and then create and activate a new adapter with the same name.
-    /// &lt;p class="Note"&gt; After deactivate returns, no new requests are processed by the object adapter.
-    /// However, requests that have been started before deactivate was called might still be active. You can
-    /// use waitForDeactivate to wait for the completion of all requests for this object adapter.
+    /// Deactivate this object adapter.
+    /// After deactivation, the object adapter stops receiving requests through its endpoints. Object adapters that
+    /// have been deactivated must not be reactivated again.
+    /// Once deactivated, it is possible to destroy the adapter to clean up resources and then create and activate a new
+    /// adapter with the same name.
+    /// After deactivate returns, no new requests are processed by the object adapter. However, requests that have been
+    /// started before deactivate was called might still be active. You can use waitForDeactivate to wait for the
+    /// completion of all requests for this object adapter.
```

Comment:
Is this true for bi-dir and collocated OAs? I thought that it would still process new requests.

Reply:
This was bogus. Rewritten.